create_aio_app/main.py

```diff
@@ -27,11 +28,14 @@ def main():
     try:
         result = cookiecutter(template_path, **kwargs)
     except OutputDirExistsException as exc:
-        print('\n\n\033[91mDirectory with such name already exists!\033[00m\n')
+        print(click.style(
```

Comment:
Use `click.echo`, or `click.fail` or `click.abort`.
Anyway this kind of info must go to stderr.